### PR TITLE
Add logging on teardown

### PR DESCRIPTION
**Pull request by aleksbykov**

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -1131,12 +1132,11 @@ def clean_resources(self):
         self.kill_stress_thread()
         db_cluster_errors = None
         db_cluster_coredumps = None
-        db_cluster_system_details = None
         if self.db_cluster is not None:
             db_cluster_errors = self.db_cluster.get_node_database_errors()
             self.db_cluster.get_backtraces()
             db_cluster_coredumps = self.db_cluster.coredumps
-            db_cluster_system_details = self.get_system_details()
+            # db_cluster_system_details = self.get_system_details()
```
Comment (bentsi) [inline sdcm/tester.py]: remove comment

File: sdcm/utils.py [hunk 2]
```diff
@@ -177,15 +177,16 @@ def remove_comments(data):
 
 class S3Storage(object):
     @staticmethod
-    def generate_url(file_path):
-        job_name = get_job_name()
+    def generate_url(file_path, testid=None):
+        prefix_path = testid if testid else get_job_name()
+        # job_name = get_job_name()
```
Comment (bentsi) [inline sdcm/utils.py]: - remove support for job_name
- upload screenshot also to the test-id dir

File: sdcm/tester.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/tester.py]: done

File: sdcm/utils.py (hunk 2 above)
Comment (aleksbykov) [reply sdcm/utils.py]: rename testid to prefix. left as parameter of method, the goal is not to import cluster to utils for one function.
screenshots uploaded to test_id dir.

**Pull request merged**